golang/cosmos/x/vstorage/vstorage.go

```diff
@@ -61,6 +61,10 @@ func (sh vstorageHandler) Receive(cctx *vm.ControllerContext, str string) (ret s
 		keeper.SetStorageAndNotify(cctx.Context, msg.Path, msg.Value)
 		return "true", nil
 
+	case "setWithoutNotify":
+		keeper.SetStorage(cctx.Context, msg.Path, msg.Value)
+		return "true", nil
+
 	case "append":
 		err = keeper.AppendStorageValueAndNotify(cctx.Context, msg.Path, msg.Value)
```

Comment:
Can `append` be triggered through chainStorage? In which case should it have a similar `appendWithoutNotify`?

Reply:
The use case enabled by `"append"` is designed to work only with notification, since `chainStorage` will need events to be present in order to follow a storage node efficiently.